etor/src/main/resources/rule_definitions.json

```
@@ -0,0 +1,23 @@
+{
+  "rules": [ {
+      "name": "requiredReceiverId",
+      "description": "Message has required receiver id",
+      "warningMessage": "Message doesn't have required receiver id",
+      "conditions": [ ],
+      "validations": [
+        "Bundle.entry.resource.ofType(MessageHeader).destination.receiver.resolve().identifier.value.exists()"
+      ]
+    },
+    {
+      "name": "requiredReceiverId",
+      "description": "ORM has required Card Number",
+      "warningMessage": "ORM doesn't have required Card Number",
+      "conditions": [
+        "Bundle.entry.resource.ofType(MessageHeader).event.code = 'O01'"
+      ],
+      "validations": [
+        "Bundle.entry.resource.ofType(Observation).where(code.coding.code = '57723-9').valueCodeableConcept.coding.code.exists()"
```

You'll need to confirm this, but FHIR Path allows you to not specify the "type" of a field.

```suggestion
        "Bundle.entry.resource.ofType(Observation).where(code.coding.code = '57723-9').value.coding.code.exists()"
```